src/locales/en.json

```diff
@@ -0,0 +1,149 @@
+    },
+    "help": {
+      "header": "Mattermost Trello Plugin - Slash Command Help",
+      "command_help": "Launch the Trello plugin command line help syntax, check out the [documentation]({{ homepageUrl }}).",
+      "command_configure": "Configure Trello workspace",
+      "command_create_description": "Create a new card",
+      "command_add_subcription": "Subscribe channel to receive notifications from a Trello board",
+      "command_list_subcription": "Show all your current subscriptions",
+      "command_stop_subcription": "Stop receiving Trello board's notifications",
+      "command_connect": "Log in to your Trello account",
+      "command_disconnect": "Log out from your Trello account"
+    },
+    "install": {
+      "description": "To finish configuring the Trello app please read the [Quick Start]({{ homepageUrl }}#quick-start) section of the README"
+    },
+    "subscription": {
+      "response_add": "Subscription will be created",
+      "response_remove": "Subscription will be deleted",
+      "response_get": "Subscription List: Found {{ count }} open subscriptions.",
+      "response_subcription": "- Subscription ID: {{ id }} - {{ description }}"
+    },
+    "webhook": {
+      "error": "Error webhook: {{ error }}",
+      "card_moved": "Card moved {{ card }} (\"{{ board }}\" Board)",
+      "board": "Board {{ name }}",
+      "text_moved": "Card \" {{ card }}\" was moved (from \" {{ listBefore }}\" list to \"{{ listAfter }}\" list)",
+      "message_created": "Card created \"{{ card }}\"  (\"{{ board }}\" Board)",
+      "text_created": "Card \"{{ card }}\" was created (in \"{{ list }}\" list)"
+    }
+  },
+  "binding": {
+    "help": {
+      "description": "Show Trello Help",
+      "title": "Show Trello Help Title"
+    },
+    "get": {
+      "description": "Add a new Card To Board"
+    },
+    "create": {
+      "decription": "Add a new Card To Board",
+      "card_name": "Card Name",
+      "card_description": "Name of the card",
+      "board_name": "Board Name",
+      "board_description": "Name of the board",
+      "list_name": "List Name",
+      "list_description": "Name of the list"
+    },
+    "subscription": {
+      "get": "Subscribe current channel to a Trello board"
+    },
+    "add": {
+      "description": "Subscribe current channel to a Trello board",
+      "title": "Subscribe channel to Trello board",
+      "board_name": "Board name",
+      "channel": "Channel"
+    },
+    "sub_list": {
+      "description": "Get list of Trello boards subscribed to current channel"
+    },
+    "remove": {
+      "description": "Remove subscription from current channel",
+      "title": "Unsubscribe from Trello board notifications",
+      "subcription": "Subscription ID"
+    },
+    "configure": {
+      "description": "Configure the installed Trello account",
+      "title": "Show Trello Help Title"
+    },
+    "account": {
+      "description": "Configure your Trello credentials",
```

Comment:
How would these strings appear to the user? Would the title and body text appear exactly the same as coded here? If so, we'll want to revisit.